atom/browser/native_window_mac.mm

```diff
@@ -470,6 +470,15 @@ - (void)drawRect:(NSRect)dirtyRect {
   return [window_ styleMask] & NSFullScreenWindowMask;
 }
 
+void NativeWindowMac::SetBounds(const gfx::Rect& bounds) {
+  Move(bounds);
```

Comment:
It seems that `NativeWindow::Move` is actually the `NativeWindow::SetBounds`, I think we can just do a function rename instead of adding a new one.